File: module-wan/src/main/java/com/huyingbao/module/wan/app/WanAppStore.java
package com.huyingbao.module.wan.app;

import android.app.Application;

import com.huyingbao.core.arch.dispatcher.RxDispatcher;
import com.huyingbao.core.arch.store.RxAppStore;

import javax.inject.Inject;
import javax.inject.Singleton;

@Singleton
public class WanAppStore extends RxAppStore {
    @Inject
    public WanAppStore(Application application, RxDispatcher rxDispatcher) {
        super(application, rxDispatcher);
    }
}

File: module-wan/src/main/java/com/huyingbao/module/wan/app/WanAppStore.kt
package com.huyingbao.module.wan.app

import android.app.Application

import com.huyingbao.core.arch.dispatcher.RxDispatcher
import com.huyingbao.core.arch.store.RxAppStore

import javax.inject.Inject
import javax.inject.Singleton

@Singleton
class WanAppStore @Inject constructor(
        application: Application,
        rxDispatcher: RxDispatcher
) : RxAppStore(application, rxDispatcher)
